src/components/editMeasure/details/MeasureDefinitions/MeasureDefinitionsValidator.ts

```diff
@@ -1,10 +1,6 @@
 import * as Yup from "yup";
 
 export const MeasureDefininitionsValidator = Yup.object().shape({
-  term: Yup.string()
-    .required("Measure Definition Term is required.")
-    .max(250, "Measure Definition Term cannot be more than 250 characters."),
-  definition: Yup.string()
-    .required("Measure Definition is required.")
-    .max(250, "Measure Definition cannot be more than 250 characters."),
+  term: Yup.string().required("Measure Definition Term is required."),
+  definition: Yup.string().required("Measure Definition is required."),
```

Comment:
We don't have any unit test cases for these validations. Do you think it would help us to add unit tests ?

Reply:
It wouldn't add any code coverage. 
These validations are done third party through Formik.

Reply:
Tests Added